.gitlab-ci.yml

```diff
@@ -75,8 +75,8 @@ test_types_and_serialize:
     - RIR_CHECK_PIR_TYPES=1 bin/gnur-make-tests check-all
     - ../../tools/check-gnur-make-tests-error
     - RIR_SERIALIZE_CHAOS=1 bin/tests
-    - RIR_SERIALIZE_CHAOS=1 bin/gnur-make-tests test-all-basics
     - RIR_SERIALIZE_CHAOS=3 bin/tests
+    - RIR_SERIALIZE_CHAOS=$(shuf -i 55-75 -n 1) bin/gnur-make-tests check-all
```

Comment:
if you do that, please also print the value to the console, so it is possible to reproduce a failure. see PIR_DEOPT_CHAOS_SEED for an example